alma/controllers/admin/AdminAlmaRefunds.php

```diff
@@ -164,4 +132,52 @@ protected function runRefund($paymentId, $amount, $isTotal)
 
         return $alma->payments->refund($paymentId, $isTotal, almaPriceToCents($amount));
     }
+
+    private function changeOrderStatusToRefund($order)
```

Comment:
```suggestion
    private function setOrdersAsRerfund($order)
```

Reply:
can you add phpdoc to tell that this method will update order / sub-orders status only if state id not already `PS_OS_REFUND` (+ `@param` & `@refund`)